example/mvc/lib/src/Model.dart

```diff
@@ -0,0 +1,70 @@
+// Copyright 2018 The Flutter Architecture Sample Authors. All rights reserved.
+// Use of this source code is governed by the MIT license that can be found
+// in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:mvc/src/todo_list_model.dart';
+
+class Model {
+  final todoModel = TodoListModel();
+
+  VisibilityFilter get activeFilter => todoModel.activeFilter;
+
+  set activeFilter(VisibilityFilter filter) {
+    todoModel.activeFilter = filter;
+  }
+
+  List<Map> get todos => todoModel.todos.map(To.map).toList();
```

Comment:
hehe Indeed.  It is confusing isn't it.

I mixed them only to demonstrate (rather clumsy it seems) that a Model could also serve as a 'conduit' and merely 'relay' the commands and data to another data source. One, in this case, uses a different a class called 'Todo' while the app's interface or View (for better or for worse) doesn't use a 'middle man' and instead uses Dart's built-in Map type.

Of course, it's not unprecedented to use Maps: [sqflite](https://pub.dartlang.org/packages/sqflite). 
```
Future<List<Map<String, dynamic>>> query(String table,
Future<List<Map<String, dynamic>>> rawQuery(String sql, [List arguments]);
```
But, yeah.  May have to rethink this.

Reply:
Gotcha. Yah, I can definitely see how conduit classes could be helpful in certain situations! In this case, I think I found it a bit confusing since it seemed to convert a Todo into a Map, which felt like a confusing transformation for the reasons listed above. 

Overall, I don't want to modify your sample too much if you're happy with it, just raise questions that I found a bit confusing and others might run into as well. If you'd like to keep this structure, it might be good to say in the README "We're just using a Map for the View instead of a Middle man class, like a TodoViewModel"